core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2228,6 +2228,15 @@ private SqlDialect nonOrdinalDialect() {
     sql(query).withBigQuery().ok(expected);
   }
 
+  @Test void testBigQueryInstrFunction() {
+    final String query = "SELECT INSTR('A', 'ABC', 1, 1) from \"product\"";
+    final String expected = "SELECT INSTR('A', 'ABC', 1, 1)\n"
+        + "FROM foodmart.product";
+    final Sql sql = fixture().withBigQuery().withLibrary(SqlLibrary.BIG_QUERY);
```

Comment:
think you can do something like sql(query).withBigQuery().withLibrary(SqlLibrary.BIG_QUERY).ok(expected); shouldn't need to create Sql object i dont think

Reply:
also if the function is valid for other libraries, could be a good idea to test with those libraries as well